pkg/controller/machine.go

```diff
@@ -793,6 +856,16 @@ func (c *controller) updateMachineStatus(
 	return clone, nil
 }
 
+// isMachineStatusEqual checks if the status of 2 machines is similar or not.
+func (c *controller) isMachineStatusEqual(s1, s2 v1alpha1.MachineStatus) bool {
+	s1Copy, s2Copy := s1.DeepCopy(), s2.DeepCopy()
+	s1Copy.LastOperation.Description, s2Copy.LastOperation.Description = "", ""
+	s1Copy.LastOperation.LastUpdateTime, s2Copy.LastOperation.LastUpdateTime = metav1.Time{}, metav1.Time{}
+	s1Copy.CurrentStatus.LastUpdateTime, s2Copy.CurrentStatus.LastUpdateTime = metav1.Time{}, metav1.Time{}
```

Comment:
Did we want to change this logic to compare the time difference between the timestamps here right?

Reply:
Hm, if the status is older than ~30mins, then update it anyways, I'll do that.